timeline-chart/src/time-graph-unit-controller.ts

```diff
@@ -1,9 +1,21 @@
 import { TimelineChart } from "./time-graph-model";
+import { BIMath } from './bigint-utils';
 import { TimeGraphRenderController } from "./time-graph-render-controller";
 
 export class TimeGraphUnitController {
+
+    
     protected viewRangeChangedHandlers: ((newRange: TimelineChart.TimeGraphRange) => void)[];
     protected _viewRange: TimelineChart.TimeGraphRange;
+    
+    /**
+     * How many extra view ranges left and right of view range.
+     * Increasing this increases world size / data cache.
+     */
+    private _worldRenderFactor = 1.5;
```

Comment:
the default value of 1.5 leads to a cache of 1.5 times the view range left and right. Shouldn't the default be 1.0?

Reply:
Set to 1.